src/Uno.UI.RuntimeTests/Tests/Microsoft_UI_Xaml_Controls/Given_TabViewItem.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Uno.Extensions.Specialized;
+
+namespace Uno.UI.RuntimeTests.Tests.Microsoft_UI_Xaml_Controls
+{
+	[TestClass]
+	[RunsOnUIThread]
+	public class Given_TabViewItem
+	{
+		[TestMethod]
+		public void When_TabViewItem_Is_Disabled()
+		{
+			var tabViewItem = new TabViewItem { IsEnabled = false };
+			Assert.IsTrue(tabViewItem.CurrentVisualStates.Contains("Disabled"));
```

Comment:
The `CurrentVisualStates` API is available in DEBUG only. Is there a better way to test?

Reply:
Create a helper in Uno.UI.RuntimeTests that does the same thing, eg VisualStatesHelper.GetCurrentStates(). Probably it will descend into the visual children of the control and find the first one that has one or more VisualStateGroups.

Reply:
You can also hook onto the [currentstatechanged](https://docs.microsoft.com/en-us/dotnet/api/system.windows.visualstategroup.currentstatechanged?view=net-5.0) event in the test control and show the value in a TextBlock.

Reply:
@jeromelaban Does this suggest doing that in UI Test like the following test?

https://github.com/unoplatform/uno/blob/ee6f9a98553c920305afc1558bd1554fde847071/src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ProgressBar/WinUIProgressBarPage.xaml.cs#L46-L49

Reply:
Precisely!